core/src/main/scala/chisel3/Assert.scala

```diff
@@ -77,6 +90,38 @@ object assert {
   }
 }
 
+object assume {
+  /** Named class for assumes. */
+  final class Assume(private[chisel3] val predicate: Bool) extends SimStatement
+
+  def apply(predicate: Bool)(
+    implicit sourceInfo: SourceInfo,
+    compileOptions: CompileOptions): Assume = {
+    val a = new Assume(predicate)
+    when (!Module.reset.asBool) {
+      val clock = Module.clock
+      Builder.pushCommand(Verification(a, Formal.Assume, sourceInfo, clock.ref, predicate.ref, ""))
+    }
+    a
+  }
+}
+
+object cover {
```

Comment:
nit: perhaps these should live in their own files, or else rename this file to not be `Assert.scala`

Reply:
It already did have the `stop` implementation in it. But yeah, renaming it to `Verification.scala` might be a good idea. (I am also happily taking any alternative naming suggestions :) )

Reply:
done